robottelo/constants.py

```diff
@@ -39,6 +39,13 @@
 }
 MAJOR_VERSION_DISTRO = {value: key for key, value in DISTROS_MAJOR_VERSION.items()}
 
+BROKER_DEPLOY_WORKFLOW = "deploy-base-rhel"
+BROKER_PROVISIONING_PROVIDER = "RHEV"
+BROKER_RHEL77 = {
+    "workflow": BROKER_DEPLOY_WORKFLOW,
+    "rhel_version": "7.7",
```

Comment:
Before this leaves draft, engage me on this @JacobCallahan . Pretty sure this will be changing relative to formatting but not function.

Reply:
I was hoping for that change to be ready :)